pkg/util/byteutil/byteutil_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2019 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package byteutil
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestUint32(t *testing.T) {

```

Comment:
no need to add this test
`Uint32ToBytes` and `Uint32ToBytesBigEndian` just calls standard golang library func

Reply:
unit test is to test the component in our codebase, and it is supposed to cover all the possible cases.
Take
```
// Uint32ToBytesBigEndian converts a uint32 to 4 bytes in big-endian
func Uint32ToBytesBigEndian(value uint32) []byte {
	bytes := make([]byte, 4)
	binary.BigEndian.PutUint32(bytes, value)
	return bytes
}
```
for example, we may make mistake in `bytes := make([]byte, 4)`, so unit test is needed to ensure we don't pass wrong parameter to `binary.BigEndian.PutUnit32`.

Even for
```
// BytesToUint64BigEndian converts 8 bytes to uint64 in big-endian
func BytesToUint64BigEndian(value []byte) uint64 {
	return binary.BigEndian.Uint64(value)
}
```
in which we call the function `binary.BigEndian.Uint64` with the identical parameters, the unit test is not for `binary.BigEndian.Uint64`, but to validate the function `BytesToUint64BigEndian` is doing what it is claiming to do.

**Conclusion**: We do need unit tests for these functions.